BIPs/2023-W26/BIP-349.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Files are not put in weekly directories or given BIP numbers until they go to snapshot. 

Please retitle your PR to start with BIP-XXX and rename the file to be called something other than BIP-###, or we will have to close it in the next day or two to not confuse our weekly governance process.

Please put it in the BIPs/00proposed directory.  The file should not be named BIP-XXX and put into a weekly directory until it has been posted to snapshot.  This is done by someone who has over 200,000 veBAL in delegation: https://docs.balancer.fi/concepts/governance/process.html#step-3-develop-and-validate-transaction-pull-request

At the moment this PR is kind of in the way(or can not be merged).

@westonnelson I would request that you close this PR, and reopen it once you have someone with the 200k veBAL to post a snapshot on a Thursday.

When so please follow the instructions above and place the file in the 00proposed directory.

Further our payload report is currently failing on your file. 

Lastly... Your forum post talks about a number of contracts and code, some of which seems quite advanced.   There are no links to any security reviews or audits.  There's also people saying they have had bad experiences engaging in your projects in the past, which go unanswered.

I think you will have an easier time finding someone with 200k veBAL to post this proposal if you spend a bit of time working on your forum post and demonstrating that your project has been reviewed, is safe, somewhat permissionless/etc.

I suggest you close this BIP for now, get the rest of your gauge proposal in a good place, and come back with a PR once other questions have been answered/sorted out in the forum.